### PR TITLE
Concept name display improvements

### PR DESCRIPTION
**Pull request by paulwarren-wk**
This PR improves the display of concept names in the fact inspector, which are often too long to fit in the available width:

image

This PR:

* Prevents line-breaking anywhere in the concept name, avoiding the split on the - in ifrs-full.
* Adds a title attribute so that the full name is available via a mouseover.
* Adds a copy icon so that the full name can be easily copied even though it may not be visible.

image

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
<details>
<summary>(1) Vulnerable direct dependencies were detected</summary>
<li>2 vulns in <code>numpy < 1.22</code> via <code>requirements.txt</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=ixbrl-viewer">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: iXBRLViewerPlugin/viewer/src/less/inspector.less [hunk 1]
```diff
@@ -708,6 +708,28 @@
       }
     }
 
+    .concept td > div {
+      display: flex;
+
+      span.text {
+        white-space: nowrap;
```
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/less/inspector.less:715]: (Arbitrary line for discussion.)

In other contexts, we have found it useful to split concept local names on heuristic word boundaries and then to insert [`<wbr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) between each word.

**Review by brettkail-wk — APPROVED** [review]

**Review by sagesmith-wf — APPROVED** [review]

File: iXBRLViewerPlugin/viewer/src/less/inspector.less (hunk 1 above)
Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/less/inspector.less:715]: That would certainly give neater wrapping, but in this case I think it's actually preferable to truncate, as we already have the standard label shown a little way above, and personally the only thing I ever want to do with the concept name is copy and paste it elsewhere.

**Review by austinmatherne-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Builds successfully. Testing locally succeeded.

@Workiva/release-management-p

**Pull request merged**